spec/unit_tests/moab/storage_object_version_spec.rb

```diff
@@ -231,58 +255,66 @@
-      "ingests/jq937jp0017/v0001/data",
-      "ingests/jq937jp0017/v0001/data/content",
-      "ingests/jq937jp0017/v0001/data/content/intro-1.jpg",
-      "ingests/jq937jp0017/v0001/data/content/intro-2.jpg",
-      "ingests/jq937jp0017/v0001/data/content/page-1.jpg",
-      "ingests/jq937jp0017/v0001/data/content/page-2.jpg",
-      "ingests/jq937jp0017/v0001/data/content/page-3.jpg",
-      "ingests/jq937jp0017/v0001/data/content/title.jpg",
-      "ingests/jq937jp0017/v0001/data/metadata",
-      "ingests/jq937jp0017/v0001/data/metadata/contentMetadata.xml",
-      "ingests/jq937jp0017/v0001/data/metadata/descMetadata.xml",
-      "ingests/jq937jp0017/v0001/data/metadata/identityMetadata.xml",
-      "ingests/jq937jp0017/v0001/data/metadata/provenanceMetadata.xml",
-      "ingests/jq937jp0017/v0001/data/metadata/versionMetadata.xml"
-    ]
+  describe '#ingest_dir' do
+    before do
+      source_dir = packages_dir.join("v0001/data")
+      temp_storage_object_version = described_class.new(temp_storage_object, 1)
+      target_dir = temp_storage_object_version.version_pathname.join('data')
+      use_links = true
+      temp_storage_object_version.ingest_dir(source_dir, target_dir, use_links)
+    end
+
+    it 'result contains expected files' do
+      files = []
+      temp_object_dir.find { |f| files << f.relative_path_from(temp_dir).to_s }
+      expect(files.sort).to eq [
+        "ingests/jq937jp0017",
+        "ingests/jq937jp0017/v0001",
+        "ingests/jq937jp0017/v0001/data",
+        "ingests/jq937jp0017/v0001/data/content",
+        "ingests/jq937jp0017/v0001/data/content/intro-1.jpg",
+        "ingests/jq937jp0017/v0001/data/content/intro-2.jpg",
+        "ingests/jq937jp0017/v0001/data/content/page-1.jpg",
+        "ingests/jq937jp0017/v0001/data/content/page-2.jpg",
+        "ingests/jq937jp0017/v0001/data/content/page-3.jpg",
+        "ingests/jq937jp0017/v0001/data/content/title.jpg",
+        "ingests/jq937jp0017/v0001/data/metadata",
+        "ingests/jq937jp0017/v0001/data/metadata/contentMetadata.xml",
+        "ingests/jq937jp0017/v0001/data/metadata/descMetadata.xml",
+        "ingests/jq937jp0017/v0001/data/metadata/identityMetadata.xml",
+        "ingests/jq937jp0017/v0001/data/metadata/provenanceMetadata.xml",
+        "ingests/jq937jp0017/v0001/data/metadata/versionMetadata.xml"
+      ]
+    end
   end
 
   describe '#ingest_file' do
-    it 'versionInventory.xml' do
-      temp_storage_object_version = described_class.new(@temp_storage_object, 2)
+    let(:temp_version_pathname) do
       temp_version_pathname = temp_storage_object_version.version_pathname
       temp_version_pathname.mkpath
-      source_file = packages_dir.join("v0002").join('versionInventory.xml')
+      temp_version_pathname
+    end
+    let(:temp_storage_object_version) { described_class.new(temp_storage_object, 2) }
 
+    it 'ingests single source files, cumulatively' do
+      # ingest versionInventory.xml
```

Comment:
cruft?

Reply:
no - trying to explain why there are two hunks in the spec (there were before ...)